ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/encoding/TransactionRLPEncoder.java

```diff
@@ -77,8 +76,6 @@ static void encodeFrontier(final Transaction transaction, final RLPOutput out) {
   }
 
   static void encodeEIP1559(final Transaction transaction, final RLPOutput out) {
-    ExperimentalEIPs.eip1559MustBeEnabled();
```

Comment:
Why do you remove that ?

Reply:
In general we're moving the transactions to not be validated at serialization time. All the checks to make sure we're not creating/deserializing transactions for a different fork are done elsewhere.